app/models/abuse_report.rb

```diff
@@ -13,6 +13,8 @@ class AbuseReport < ApplicationRecord
 
   default_scope -> { order created_at: :desc }
 
+  after_create :send_confirmation
```

Comment:
Zusammen mit `set_confirmation` inkl. zugehöriger Callbackdefinition und `send_confirmation` in ein `ConfirmationCallbacks` Concern auslagern und an allen Stellen verwenden / inkludieren.